src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+  };
+
+  calculator.add = function(number) {
+    calculator.result += number;
+  };
```

Comment:
better to make method fields directly in calculator object, and after that will be possible to use `this` keyword
```suggestion
  const calculator = {
    result: 0,
    
    add(number) {
       this.result += number;
    };
    
    // same for other methods
    
     operate(callback, number) {
        callback.call(this, number);

      return this;
    },
  };
```
and in this case use call method for calling callback in operate method